handlers/events_test.go

```diff
@@ -0,0 +1,70 @@
+package handlers
+
+import (
+	"github.com/golang/mock/gomock"
+	"github.com/hashicorp/serf/serf"
+	"github.com/julienschmidt/httprouter"
+	mockcluster "github.com/livepeer/catalyst-api/mocks/cluster"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+func TestEventHandler(t *testing.T) {
+	require := require.New(t)
+
+	tests := []struct {
+		requestBody  string
+		wantHttpCode int
+	}{
+		{
+			requestBody: `{
+				"resource": "stream",
+				"playback_id": "123456789"
+			}`,
+			wantHttpCode: 200,
+		},
+		{
+			requestBody:  "",
+			wantHttpCode: 400,
+		},
+		{
+			requestBody:  "invalid payload",
+			wantHttpCode: 400,
+		},
+		{
+			requestBody: `{
+				"resource": "stream"
```

Comment:
I know it will fail but might wanna add a test for different `resource` values as well

Reply:
Added in c5331d91362ad527de44f026d0406dd7f16b62f5